pkgs/development/tools/misc/libtool/libtool2.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, fetchpatch, autoconf, automake, m4, perl, help2man
+{ lib, stdenv, fetchurl, fetchpatch, m4
```

Comment:
```suggestion
{ lib, stdenv, fetchurl, fetchpatch, m4
# autoconf, automake, perl, and help2man were removed here in
# https://github.com/NixOS/nixpkgs/pull/210118/ in order to speed up the stdenv bootstrap.
# Please don't add them back without checking the impact there first.

```

Reply:
The GNU tools' mechanism for avoiding rebuilds of documentation has been very fragile for at least a decade.  I'm sure it will break again, and we want to ensure that whoever does the next version bump doesn't just toss the dependencies back in as the path of least resistance.